### PR TITLE
Pending transaction was added to /latest-transactions API

### PR DESCRIPTION
**Pull request by waqarRns**

### CODE REVIEW

File: src/modules/storage/LedgerStorage.ts [hunk 1]
```diff
@@ -3442,15 +3442,44 @@ export class LedgerStorage extends Storages {
...
+                    SELECT
+                        (SELECT IFNULL(MAX(height), 0) AS height FROM blocks) AS block_height,
+	                    T.tx_hash,
+                        T.time as time_stamp,
+	                    T.tx_fee,
+                        T.tx_size,
+                        T.type,
+	                    Sum(IFNULL(O.amount,0)) as amount
+	                FROM
+	                    tx_outputs O
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts:3465]:
```suggestion
	                    tx_output_pool O
```

File: src/modules/storage/LedgerStorage.ts [hunk 2]
```diff
@@ -3442,15 +3442,44 @@ export class LedgerStorage extends Storages {
...
+                    amount,
+                    count(*) OVER() AS full_count
+                FROM(
+                    SELECT
+                        (SELECT IFNULL(MAX(height), 0) AS height FROM blocks) AS block_height,
+	                    T.tx_hash,
+                        T.time as time_stamp,
+	                    T.tx_fee,
+                        T.tx_size,
+                        T.type,
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts:3461]: '0' as status,
// Pending

File: src/modules/storage/LedgerStorage.ts [hunk 3]
```diff
@@ -3442,15 +3442,44 @@ export class LedgerStorage extends Storages {
...
+		                INNER JOIN transaction_pool T ON (T.tx_hash = I.tx_hash)
+                    UNION
+                    SELECT
+                        T.block_height,
+                        T.tx_hash,
+                        B.time_stamp,
+                        T.tx_fee,
+                        T.tx_size,
+                        T.type,
+                        Sum(IFNULL(O.amount,0)) as amount
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts]: '1' as status,
// Confirmed

**Review by TrustHenry — CHANGES_REQUESTED** [review]

File: src/modules/storage/LedgerStorage.ts (hunk 1 above)
Comment (waqarRns) [reply src/modules/storage/LedgerStorage.ts:3465]: we can not use tx_output_pool. actully we need 
tx_outputs S
                INNER JOIN tx_input_pool I ON (I.utxo = S.utxo_key)

File: src/modules/storage/LedgerStorage.ts [hunk 4]
```diff
@@ -3442,15 +3442,48 @@ export class LedgerStorage extends Storages {
...
+                    UNION
+                    SELECT
+                        T.block_height,
+                        T.tx_hash,
+                        B.time_stamp,
+                        T.tx_fee,
+                        T.tx_size,
+                        T.type,
+                        Sum(IFNULL(O.amount,0)) as amount,
+                        'Confirmed' as status
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts:3478]: How about coding with multilingual processing on the screen?

Comment (waqarRns) [reply src/modules/storage/LedgerStorage.ts:3478]: we cannot apply multilingual processing on dynamic data.
i think we should only apply it to static data.

**Review by TrustHenry — APPROVED** [review]

**Pull request merged**